lib/firmata.js

```diff
@@ -686,11 +687,15 @@ function Board(port, options, callback) {
             this.pins.push({supportedModes: [], analogChannel: analogChannel});
           }
         }
-        ready();
+        setTimeout(function() {
+          ready();
+        }, settings.firstWriteTimeout);
```

Comment:
I agree, and I'm not willing to add an entire 2 seconds on top of the existing time